src/main/java/com/wolox/training/models/Users.java

```diff
@@ -0,0 +1,95 @@
+package com.wolox.training.models;
+
+import com.sun.istack.NotNull;
+import com.wolox.training.exceptions.BookAlreadyOwnedException;
+import java.time.LocalDate;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import javax.persistence.CascadeType;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.JoinTable;
+import javax.persistence.ManyToMany;
+
+@Entity
+public class Users {
```

Comment:
Change Users to User

Reply:
Hi Mati, thanks for the tips, I pushed the changes!
With reference this last comment, the trello card indicates that it's recommended not to user the name "User" because it can bring problems with Postgres. I can use @Table anotation to define the table name but y prefer keep names equal. What do you think?

Reply:
Okeyyy, thanks!, I didnt remember this.

It's ok for me